contracts/libs/Types.sol

```diff
@@ -26,12 +26,10 @@ library Types {
     struct Batch {
         bytes32 stateRoot;
         bytes32 accountRoot;
-        bytes32 depositTree;
```

Comment:
I'm super uncertain about this. Haven't given it a good thought

Reply:
Yeah we dont want to remove that, we need it when we do rollback, all we want to do is remove [this](https://github.com/thehubbleproject/RedditHubble/blob/master/contracts/rollup.sol#L113) line as it will pick up the default zero value automatically

Reply:
Ah ah I see.